css-transitions-2/Overview.bs

```diff
@@ -167,6 +167,78 @@ Each time a new transition is generated, the current value of the (already
 incremented) <a>current transition generation</a> is stored as the
 transition's <dfn>transition generation</dfn>.
 
+## Defining [=before-change style=]: the ''@starting-style'' rule
+
+Transitions can only start during a [=style change event=] for elements which
+has a defined [=before-change style=] established by the previous
+[=style change event=], which means that newly inserted elements or elements
+that change from not [=being rendered=] to being rendered do not start
+transitions. Still, doing transitions starting from the initial rendering is a
+valid use case.
```

Comment:
This is making factual statements that become untrue as a result of the addition of `@starting-style`.  I think it would be useful to more clearly separate (a) a summary of the rules as of CSS-TRANSITIONS-1 and (b) a description of how this specification (a delta spec) modifies those rules.

Reply:
Not sure exactly how you'd want it. Tried to improve it.